viral-ngs-fasta-fetcher/src/viral-ngs-fasta-fetcher.sh

```diff
@@ -7,7 +7,24 @@ main() {
     dx cat "$resources" | tar zx -C /
 
     # Write combined fasta to /genome.fasta
-    viral-ngs/ncbi.py fetch_fastas_and_feature_tables $user_email ./ $accessions --combinedGenomeFilePrefix genome --removeSeparateFastas
+
+    # there is a change to the interface for fetching fastas, feature tables, and full GenBank records
+    # we need to check which commands are present and call them as appropriate.
+    # since argparse returns an error code if a command is missing, and exits zero
+    # if a command is present (even if arguments are missing), we can use exit codes
+    # to determine the correct functions to call
+
+    if viral-ngs/ncbi.py fetch_fastas_and_feature_tables &> /dev/null ; then
```

Comment:
@tomkinsc

I re-wrote the checks for for command return values because the applet bash script has `-e -o pipefail` set and will fail with an AppInternalError if we expose the non-zero return status.